.abigenrc

```diff
@@ -1 +1 @@
-v1.10.25
+v1.10.16
```

Comment:
Why is this diff here?

Reply:
downgraded to a version of abigen Matt recommended me (bindings weren't matching those of CI), which was eventually saved to the file after running `pnpm upgrade:abigen` in repo root. can revert back though

Reply:
Would prefer to keep the same version of abigen if possible. If it results in 100% issues with matching bindings then we can figure out why. This sometimes happened to me with weth9

Reply:
I imagine this was a mismatch with the foundry version. Try going back to the old version of abigen, and `p update:foundry` in the monorepo root.